io.github.randovania.Randovania.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.randovania.Randovania
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [ org.freedesktop.Sdk.Extension.mono6 ]
```

Comment:
I'm a little bit lost here. It seems to be a Python project but you pull in a Mono SDK extension here?

Reply:
It's a PySide6 app, but it embeds some C# applications we call via subprocess.